fairlearn/datasets/__init__.py

```diff
@@ -5,11 +5,15 @@
 """This module contains datasets that can be used for benchmarking and education."""
 
 
+from ._fetch_acsincome import fetch_acsincome
+from ._fetch_acspubliccoverage import fetch_acspubliccoverage
```

Comment:
I had missed that in #1005 @kspieks @britneyting perhaps we should add underscores between these? 
`fetch_acs_income` and `fetch_acs_public_coverage`
The only other dataset so far with multiple words is `fetch_bank_marketing`. Definitely more readable IMO.

Reply:
@romanlutz This change was in my PR. You didn't mess anything :) It's just that @britneyting's PR is based on mine so until mine is merged in, everything related to acsincome is present in her PR